Cargo.toml

```diff
@@ -15,8 +15,11 @@ keywords = ["random", "rng"]
 categories = ["algorithms"]
 
 [features]
-i128_support = []
-nightly = ["i128_support"]
+bound = []         # std::collections is unstable in 1.0.0
+duration = []      # Duration is unstable in 1.0.0
+ip_addr = []       # IpAddr is unstable in 1.0.0
```

Comment:
I'm not so keen on adding a bunch of conditional features here

Reply:
Agreed! These should be added properly with a rustc version bump.